### PR TITLE
Fix bug: Fixed a problem that was overwritten and not inputted when entering Korean

### PR DESCRIPTION
**Pull request by 2h-kim**
The part where Korean is separated and entered in CVAT standard workspace has been fixed in PR (https://github.com/opencv/cvat/pull/6916).

But in case of `Attribuite annotation`  workspace, it confirmed that the same problem still occurred and corrected it similar to the PR above.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by 2h-kim** [comment]
Below is my test case when i using korean annotation in `Attribuite annotation` workspace.
I'm unsure how to write test codes for the UI, so I'm attaching the results of the tests conducted on the UI below.

**Before**
Korean_AS_IS


**After**
Korean_TO_BE

File: CHANGELOG.md [hunk 1]
```diff
@@ -100,6 +100,9 @@ longer accepted automatically. Instead, the invitee can now review the invitatio
 - Updating cloud storage attached to CVAT using Azure connection string
   (<https://github.com/opencv/cvat/pull/7336>)
 
+- Can not input Korean correctly in text attributes in workspace Attribuite annotation
```
Comment (bsekachev) [inline CHANGELOG.md:103]: Please, do not change CHANGELOG.md

Use ``scriv create --edit`` to create CHANGELOG fragment in `changelog.d` directory

Comment (2h-kim) [reply CHANGELOG.md]: Thank you for introducing me to the `scriv create --edit` command. 
This was my first time using it to generate a changelog fragment in the changelog.d directory, and I must say, it's a great approach. 
I appreciate your guidance and am grateful for this new insight! :)

**Comment by bsekachev** [comment]
Hello

Thanks for the contribution, I did minor changes, so, please the PR is still working for you.

**Comment by bsekachev** [comment]
And please, check in PR description:  `I submit my code changes under the same [MIT License](https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.`

Otherwise we will not be able to accept it.

**Comment by 2h-kim** [comment]
@bsekachev Thanks for update pull request. 
I tried again and made sure it worked fine

**Pull request merged**